jekyll/content/terms/neural-turing-machine-ntm.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Neural Turing Machine (NTM)
+related_terms:
+ - recurrent-neural-network
+ - neural-network
+ - long-short-term-memory
+references:
+ - "[Neural Turing Machine - Wikipedia](hhttps://en.wikipedia.org/wiki/Neural_Turing_machine)"
+ - "[Neural Turing Machines](https://arxiv.org/pdf/1410.5401.pdf)"
+---
+
+Neural Turing Machines (NTM) are a differentiable computer, an consists of a RNN (commonly with LSTM), and a memory bank, where the neural network can make write and read operations. By making each operation of the NTM differentiable, it can be trained efficiently trained with gradient descent. 
```

Comment:
I would say that Neural Turing Machines are a type of recurrent neural network, instead of describing them as a computer.

The [Wikipedia page for Neural Turing Machines](https://en.wikipedia.org/wiki/Neural_Turing_machine) describes them in a similar way.